en/components/general/update-guide.md

```diff
@@ -108,55 +108,106 @@ For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from th
             $background: red
         );
-    
+
         $my-outlined-button: igx-button-theme(
             $border-color: red
         );
-    
+
         .my-raised-btn {
             @include igx-css-vars($my-raised-button);
          }
-    
+
         .my-outlined-btn {
             @include igx-css-vars($my-outlined-button);
         }
         ```
     As you can see, since the `igx-button-theme` params now have the same names for each button type, we have to scope our button themes to a CSS selector in order to have different colors for different types.
-    
-    Here you can see all the [available properties](https://www.infragistics.com/products/ignite-ui-angular/docs/sass/latest/index.html#function-igx-button-theme) of the `igx-button-theme` 
+
+    Here you can see all the [available properties](https://www.infragistics.com/products/ignite-ui-angular/docs/sass/latest/index.html#function-igx-button-theme) of the `igx-button-theme`
 
     * The `igx-typography` mixin is no longer implicitly included with `igx-core`. To use our typography styles you have to include the mixin explicitly after `igx-core` and before `igx-theme`:
 
     ```scss
     // in styles.scss
-    
+
     @include igx-core();
-    
+
     @include igx-typography(
         $font-family: $material-typeface,
         $type-scale: $material-type-scale
     );
-    
+
     @include igx-theme();
     ```
 
     > [!IMPORTANT]
     > The `igx-core` mixin should always be included first.
 
-    For each theme included in Ignite UI for Angular we provide specific `font-family` and `type-scale` variables which you can use: 
+    For each theme included in Ignite UI for Angular we provide specific `font-family` and `type-scale` variables which you can use:
 
     | **Theme** | **Font Family** | **Type Scale** |
     |----------------|-----------------|-----------------|
     | Material | $material-typeface | $material-type-scale |
     | Fluent | $fluent-typeface | $fluent-type-scale |
     | Bootstrap | $bootstrap-typeface | $bootstrap-type-scale |
     | Indigo | $indigo-typeface | $indigo-type-scale |
- 
+
+### `IgxBottomNav` and `IgxTabs`
+The two components were completely refactored in order to provide more flexible and descriptive way to define tab headers and contents. It is recommended that you update via **ng update** in order to migrate the existing **igx-tabs** and **igx-bottom-nav** definitions to the new ones.
+
+* Template
+    * The new structure defines tab items each wrapping a header and a content. The headers may contain a label and/or an icon as well as any other custom content.
+    * For header styling purposes we introduced four new directives, namely `IgxBottomNavHeaderLabel`, `IgxBottomNavHeaderIcon`, `IgxTabHeaderLabel` and `IgxTabHeaderIcon`.
+    * Since the header component now allows adding any content, the `IgxTab` directive, which was previously used to retemplate the tab's header, was removed because it is no longer necessary.
+
```

Comment:
Just mention about routerLink directive - it's now set on the header but migrations should update it.